cms/templates/js/add-list-item.underscore

```diff
@@ -0,0 +1,3 @@
+<button class="action action-add">
```

Comment:
Why does this need its own template? It seems a bit out of place. I thought maybe it was used in multiple places, but I could only see one.